pkg/monitor/cluster/cluster.go

```diff
@@ -33,13 +34,14 @@ type Monitor struct {
 	oc   *api.OpenShiftCluster
 	dims map[string]string
 
-	restconfig *rest.Config
-	cli        kubernetes.Interface
-	configcli  configclient.Interface
-	maocli     machineclient.Interface
-	mcocli     mcoclient.Interface
-	m          metrics.Emitter
-	arocli     aroclient.Interface
+	restconfig  *rest.Config
+	cli         kubernetes.Interface
+	configcli   configclient.Interface
+	maocli      machineclient.Interface
+	mcocli      mcoclient.Interface
+	m           metrics.Emitter
+	arocli      aroclient.Interface
+	operatorcli operatorclient.Interface
```

Comment:
Instead of adding the operatorclient, can we start leveraging the generic client which can query by API type?  This saves us from creating an interface per API version per cluster.  

Example: https://github.com/Azure/ARO-RP/blob/master/pkg/operator/controllers/imageconfig/image_controller.go#L44C1-L44C19

Reply:
Done. I also refactored the other calls in the monitor to use the generic client.